linting/lint-csv.py

```diff
@@ -0,0 +1,35 @@
+import os
+import subprocess
+import sys
+
+
+CSV_PATH = os.path.join(os.path.dirname(__file__), '..')
+
+# This is the default csvclean message
+# when there are no errors
+NO_ERRORS_MSG = b'No errors.\n'
+
+
+def main():
+    is_error = False
+    for filename in os.listdir(CSV_PATH):
+        if filename.endswith('.csv'):
```

Comment:
You can use reverse condition to avoid deep nesting which is less readable.
```suggestion
        if not filename.endswith('.csv'):
            continue
```

Reply:
(or you could've used `glob` to match only csv files)

Reply:
@webknjaz, these latter 2 suggestions would cause issues if they were just accepted without modifying other code. They may not be a great fit for github suggestions.

Reply:
true. no worries